internal/resources/templates/bids_results.go

```diff
@@ -64,6 +66,20 @@ const BidsResults = `
 	<div>Size: {{ .Summary.Size }}</div>
 {{ end }}
 		</div>
+		<div class="column" style="width:35%">
```

Comment:
Please switch the order of "content" and "history" `div` and change the width to 80% content and 20% history.